.github/workflows/test_and_release.yml

```diff
@@ -16,6 +16,11 @@ jobs:
     runs-on: ubuntu-20.04
     timeout-minutes: 10
     steps:
+      - name: Cancel Previous Runs
```

Comment:
Will it cancel all jobs or just this one?

Reply:
It will cancel all previous jobs and leave the current job running. Since the action runs every time, in nearly all cases (unless you push twice in <10s) there will only be one job running per branch, because the first thing it does is stop previous runners.

Reply:
Got it! Thank you.